packages/common/src/types/field.ts

```diff
@@ -123,6 +123,15 @@ export type TableCalculation = {
     format?: TableCalculationFormat;
 };
 
+export interface TableCalculationFilter extends Field {
```

Comment:
I don't think TableCalculationFilter should to extend Field, they are different types, a filter should not be a field.

Reply:
WHy do we need this type ? FilterableItem already supports TableCalculation

Reply:
it conflicts some where in the code i remember if i only use FilterableItem in the code, let me figure this out and come back to you.